.github/workflows/ci.yml

```
@@ -24,19 +24,13 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
     
-    - name: Add GOPATH/bin to PATH
-      # temporary fix
-      # see https://github.com/actions/setup-go/issues/14
-      run: |
-        set -euo pipefail
-        echo "##[set-env name=GOPATH;]$(dirname $GITHUB_WORKSPACE)"
-        echo "##[add-path]$(dirname $GITHUB_WORKSPACE)/bin"
-    
     - name: Install dependencies
       run: go mod download
     
     - name: Install gox
-      run: hack/install-gox.sh
+      run: | 
+        export PATH=${PATH}:`go env GOPATH`/bin
```

```suggestion
        export PATH=${PATH}:"$(go env GOPATH)/bin"
```

Please also quote. Just for good measure :)

---

> Please also quote. Just for good measure :)

Thanks, @corneliusweig I will keep it in mind :)